.changeset/org-ownershipcard-filteradded.md

```diff
@@ -1,6 +1,6 @@
 ---
-"example-app": patch
-"@backstage/plugin-org": patch
+'example-app': patch
```

Comment:
Don't need to bump `example-app` as that package is not published.

Reply:
ok, thanks. Fixed.